src/plugins/imageLink/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "ImageLink",
+    description: 'Suppresses the hiding of links for "simple embeds"',
```

Comment:
this description is hard to understand imo. maybe something like
```suggestion
    description: "Never hide image links in messages, even if it's the only content",
```
?